src/composables/multisigAccountCreate.ts

```diff
@@ -118,39 +128,42 @@ export function useMultisigAccountCreate({ store }: IDefaultComposableOptions) {
    * First step of creating the multisig account
    * Prepare multisig account creation transaction
    */
-  async function prepareVaultCreationRawTx(payerId: string, accountId: string) {
-    if (!pendingMultisigCreationTxs.value[accountId]?.signedAttachTx) {
+  async function prepareVaultCreationRawTx(
```

Comment:
I suggest to add empty line before this one.

Reply:
I didn't understand where.